src/js/components/Diagram/doc.js

```diff
@@ -3,6 +3,23 @@ import { describe, PropTypes } from 'react-desc';
 import { colorPropType } from '../../utils/prop-types';
 import { getAvailableAtBadge } from '../../utils/mixins';
 
+const ANIMATION_PROP = PropTypes.oneOfType([
```

Comment:
Naming recommendation to match with other `prop-types` in utils: `animationPropType`